packages/engine-rn-windows/src/tasks/task.rnv.deploy.ts

```diff
@@ -1,14 +1,21 @@
-import { Constants, Logger, PlatformManager, TaskManager } from 'rnv';
+import {
+    logErrorPlatform,
+    logTask,
+    WINDOWS,
+    XBOX,
+    PARAMS,
```

Comment:
is rnv exporting everything on top level now?

Reply:
yes. with TS it it much easier to sanitize it. for sub exports we should look into subpath approach like import { xxx } from 'rnv/submodule' in the future